pkg/policy/api/rule.go

```diff
@@ -16,6 +16,8 @@ package api
 
 import (
 	"github.com/cilium/cilium/pkg/labels"
+
+	"k8s.io/apimachinery/pkg/types"
```

Comment:
This is a no-go. We can't depend on k8s on our API.

Reply:
Too late?

```
$ git grep k8s pkg/policy/api/selector.go
pkg/policy/api/selector.go:     metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
...
```

Reply:
I don't understand why this is a no-go - why can't we depend on K8s?

Reply:
@joestringer I spoke too son. It's fine since `k8s.io/apimachinery/pkg/types` does not import anything else.